src/corefx/System.Globalization.Native/collation.cpp

```diff
@@ -407,6 +401,24 @@ const UCollator* GetCollatorFromSortHandle(SortHandle* pSortHandle, int32_t opti
     return pCollator;
 }
 
+extern "C" int32_t GlobalizationNative_GetSortVersion(SortHandle* pSortHandle)
+{
+    UErrorCode err = U_ZERO_ERROR;
+    const UCollator* pColl = GetCollatorFromSortHandle(pSortHandle, 0, &err);
+    int32_t result = 0;
+
+    if (U_SUCCESS(err))
```

Comment:
In what situations will this fail?  Just wondering when the fallback will be used and what implications that'll have.